.gitignore

```diff
@@ -5,3 +5,4 @@ packages/*/example/*.js*
 *.map
 .nx/
 packages/matrix-identity-server/matrix-server/synapse-data
+.vscode
```

Comment:
I'm not forced to use VS code and we likely do not want to add and maintain one line per personnal preference.

Generally editor settings goes onto personal global gitignores not onto the project ones that generally only excludes build materials.

CF https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
You're write I'll fix that
Thanks !

Reply:
> You're write

You're right ;-)